resources/assets/js/components/privacy-info/PrivacyInfo.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <privacy-info-logic>
+        <div class="block" v-if="toShow" slot-scope="{ toShow, accept }">
+            <div class="message">
+                Dostosowaliśmy naszą Politykę Prywatności do nowych wymogów rozporządzenia Unii Europejskiej o ochronie
+                danych osobowych.
+                <br>
+                Dalsze korzystanie z witryny zakłada, że akceptujesz naszą
+
+                <a v-if="privacyPolicyUrl" :href="privacyPolicyUrl">
```

Comment:
Can we also handle the case where there is no privacy policy URL is provided so that the text will still make sense?

Reply:
Shall I put just the text from the link label instead of the link when no URL is given?

Reply:
Yeah, I think that this is a good idea.